### ISSUE TITLE
Add to your concept

### ISSUE THREAD

**Issue by mspianos** [Makerr-Studio/CentrifugalDustSeparator-#17]
Hello,

I don't know where to contact you so I put my message here, I will remove it if it is not in the right place.

I thought last night, I know I should sleep, to add the engine, the filter and a third tank directly in your design. It is just a schematic.

Do you think that the idea is to be further explored, or should I go to sleep?
![Capture d’écran 2024-10-21 161143](https://github.com/user-attachments/assets/55301b83-c7d8-46aa-a42e-010332e6d09c)

**Comment by Makerr-Studio** [comment]
Sleep is good but I like your way of thinking! 

I doubt whether such a third phase is efficient. I think the tube diameter needs to be smaller to achieve the higher speeds and separate the very fine particles.

I'm looking into a slightly different option to place a NanoMax style cartridge filter + motor on the side of the second bin (not in line). I don't have the space above my planer for the available length. And I try to make more use of gravity in the last filter so that the fine particles spin arount the filter and fall down the side of the motor so that it can be easily emptied.

We discuss this kind of thing more on my Patreon. And we've set up a Discort server for ideas like this you can join

**Comment by mspianos** [comment]
I don't know Patreon, it seems to be paid, right?
For Discord, do you have a link to your server to give me?

**Comment by Makerr-Studio** [comment]
Yes Patreon is a way to support me and you get my support and that from the community back. I will update on my progress and share exclusive test files to print yourself. When you become a tier member the Discord server is automatically released for you.

Github is the free option to contribute! Only I check that twice a week. Patreon I watch and react on a daily basis.

**Comment by Nicokirr** [comment]
@mspianos , I'm thinking about same design, as the camvac tank is useless if all dust is captured before. Also, you can purchase vacuum motors for around 40€ each, much cheaper than the camvac. 
Did you made some progress on your idea ? 

Another reflection is on the placement of HEPA filters. They could be after the motor, to ease replacement / cleaning.
